### PR TITLE
Method(Test) names typo and lower camel case fix

### PR DESCRIPTION
**Pull request by Johnett**

### CODE REVIEW

File: wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt [hunk 1]
```diff
@@ -707,7 +707,7 @@ class SchemaTest {
   }
 
   @Test
-  fun messsageAndExtensionNameCollision() {
+  fun messageAndExtensionNameCollision() {
```
Comment (JakeWharton) [inline wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt]: This is the only one I agree with. The others don't seem incorrect in any meaningful way.

Comment (Johnett) [reply wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt]: Yes, you are right about this one. And the other three I think I have missed some details(not typo issue. it's lower camel case issue(I have changed the title)). I have commented under each correction please check.

File: wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt [hunk 2]
```diff
@@ -542,7 +542,7 @@ class PrunerTest {
   }
 
   @Test
-  fun namespacedExtensionFieldsAreRetained() {
+  fun namespaceExtensionFieldsAreRetained() {
```
Comment (Johnett) [inline wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt]: We can ignore this one it's a misunderstanding from my end.

File: wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt [hunk 3]
```diff
@@ -399,7 +399,7 @@ class SchemaTest {
   }
 
   @Test
-  fun oneofFieldTypeUnknown() {
+  fun oneOfFieldTypeUnknown() {
```
Comment (Johnett) [inline wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt]: Here the method name is not in proper lower camel case unless if we consider "oneof" as a single word(I have some proofs please see below).
In some part of the class, the word("oneof") is used in perfect lower camel case(`fun oneOfGroupsThrow()`,`fun duplicateTagValueDisallowedInOneOf()`). Incase these changes are irrelevant I will change the pull request.

File: wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt [hunk 4]
```diff
@@ -849,7 +849,7 @@ class SchemaTest {
   }
 
   @Test
-  fun oneofLabelDisallowed() {
+  fun oneOfLabelDisallowed() {
```
Comment (Johnett) [inline wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt]: Lower camel case issue.

File: wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt (hunk 2 above)
Comment (oldergod) [reply wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt]: Let's revert this one. I don't care about the rest.

Comment (Johnett) [reply wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt]: reverted.

**Comment by oldergod** [comment]
Could you sign [our CLA](https://docs.google.com/forms/d/e/1FAIpQLSeRVQ35-gq2vdSxD1kdh7CJwRdjmUA0EZ9gRXaWYoUeKPZEQQ/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1) so we can merge your PR?

**Comment by Johnett** [comment]
I have signed the CLA.

**Pull request merged**